minimint/src/net/api.rs

```diff
@@ -126,6 +126,18 @@ fn server_endpoints() -> &'static [ApiEndpoint<MinimintConsensus<rand::rngs::OsR
                 Ok(tx_status)
             }
         },
+        api_endpoint! {
+            "/list_gateways",
+            async |minimint: &MinimintConsensus<rand::rngs::OsRng>, _v: serde_json::Value| -> Vec<LightningGateway> {
```

Comment:
```suggestion
            async |minimint: &MinimintConsensus<rand::rngs::OsRng>, _: ()| -> Vec<LightningGateway> {
```

Reply:
Suggested change doesn't compile w/ error:

```
async |minimint: &MinimintConsensus<rand::rngs::OsRng>, _: ()| -> Vec<LightningGateway> {
                                                        ^ no rules expected this token in macro call
```

Reply:
ah `_` is not supported by the macro
you can just try `_v`

Reply:
Updated with `_v: ()`